client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -141,109 +140,105 @@ const MarketplaceThankYou = () => {
+							</MarketplaceHeaderTitle>
+						</MarketplaceThankyouSection>
+						<MarketplaceThankyouSection>
+							<MarketplaceHeaderTitle
+								subtitle
+								className="marketplace-thank-you__body-header wp-brand-font"
+							>
+								{ translate( 'What’s next?' ) }
+							</MarketplaceHeaderTitle>
+							<MarketplaceNextSteps>
+								<h3>{ translate( 'Plugin setup' ) }</h3>
+								<div>
+									<p>
+										{ translate(
+											'Get to know Yoast SEO and customize it, so you can hit the ground running.'
+										) }
+									</p>
+									<div>
+										<FullWidthButton
+											href={ yoastSeoPageUrl }
+											primary
+											busy={ isRequestingMenu }
+											// TODO: Menu links are not properly loading on initial request, post transfer so yoastSeoPageUrl will remain empty post transfer
+											// This should be fixed with perhaps a work around to periodically poll for the menu with various domains until it loads
+											// or maybe blocking the user from entering this flow until a domain acquires SSL
+											disabled={ ! yoastSeoPageUrl }
+										>
+											{ translate( 'Get started' ) }
+										</FullWidthButton>
+									</div>
+								</div>
+								<h3>{ translate( 'Start putting SEO to work' ) }</h3>
+								<div>
 									<p>
 										{ translate(
-											'Our Happiness Engineers are here if you need help, or if you have any questions.'
+											"Improve your site's performance and rank higher with a few tips."
 										) }
 									</p>
-									<VerticalNav>
-										<VerticalNavItem path={ '/help/contact' }>
-											{ translate( 'Ask a question' ) }
-										</VerticalNavItem>
-										<VerticalNavItem path={ 'https://wordpress.com/support' }>
-											{ translate( 'Support documentation' ) }
-										</VerticalNavItem>
-									</VerticalNav>
-								</MarketplaceThankyouSection>
-							</div>
-						</ThankYouBody>
-					</MarketplaceThankYouContainer>
-				</>
-			) }
+									<div>
+										<FullWidthButton
+											href={ postsPageUrl }
+											busy={ isRequestingMenu }
+											disabled={ ! yoastSeoPageUrl }
+										>
+											{ translate( 'View posts' ) }
+										</FullWidthButton>
+									</div>
+								</div>
+							</MarketplaceNextSteps>
+						</MarketplaceThankyouSection>
+						<MarketplaceThankyouSection>
+							<MarketplaceHeaderTitle
+								subtitle
+								className="marketplace-thank-you__body-header wp-brand-font"
+							>
+								{ translate( 'How can we help?' ) }
+							</MarketplaceHeaderTitle>
+							<p>
+								{ translate(
+									'Our Happiness Engineers are here if you need help, or if you have any questions.'
+								) }
+							</p>
+							<VerticalNav>
+								<VerticalNavItem path={ '/help/contact' }>
+									{ translate( 'Ask a question' ) }
+								</VerticalNavItem>
+								<VerticalNavItem path={ 'https://wordpress.com/support' }>
```

Comment:
Nit: this should be `localizeUrl( 'https://wordpress.com/support')`. It won't make any difference for now, but I felt like to make use of this chance to let you know about this i18n practice :)

Reply:
And just in case we don't want to localise it (which I don't see why not FWIW), we don't need to phrase it as a JS expression, rather it can be a string:

```suggestion
								<VerticalNavItem path="https://wordpress.com/support">
```

Reply:
Added the localizeUrl part.